src/Verifier/SAW/Prim.hs

```diff
@@ -280,6 +281,7 @@ instance Show EvalError where
   show e = case e of
     InvalidIndex i -> "invalid sequence index: " ++ show i
     DivideByZero -> "division by 0"
+    UnsupportedPrimitive b p -> "unsupported primitive " ++ p ++ " in backend" ++ b
     UserError msg -> "Run-time error: " ++ msg
```

Comment:
This is totally a nitpick, but I think it'd be more readable if this was: `"unsupported primitive " ++ p ++ " in " ++ b ++ " backend"`.

Reply:
👍